oai-pmh-handler/src/main/java/no/sikt/nva/oai/pmh/handler/OaiPmhHandler.java

```diff
@@ -0,0 +1,44 @@
+package no.sikt.nva.oai.pmh.handler;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.google.common.net.MediaType;
+import jakarta.xml.bind.JAXBElement;
+import java.net.HttpURLConnection;
+import java.util.List;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.Environment;
+import org.openarchives.oai.pmh.v2.OAIPMHtype;
+
+public class OaiPmhHandler extends ApiGatewayHandler<Void, String> {
+  private final XmlSerializer<JAXBElement<OAIPMHtype>> xmlSerializer = new JaxbXmlSerializer();
+  private final OaiPmhDataProvider dataProvider = new DefaultOaiPmhDataProvider();
+
+  public OaiPmhHandler(Environment environment) {
+    super(Void.class, environment);
+  }
+
+  @Override
+  protected void validateRequest(Void unused, RequestInfo requestInfo, Context context)
+      throws ApiGatewayException {
+    // no-op
```

Comment:
We likely have a strong opinion about what it means to validate such a request. Probably its own task

Reply:
Agreed. This is just scaffolding.